env/patcheval/rule_evaluator.py

```
@@ -0,0 +1,191 @@
+            logger=logging.getLogger(f"safactory.patcheval.official.{cve_id.lower()}"),
+            cve=cve_id,
+        )
+        poc_passed, poc_log, unit_tests_passed, unit_test_log, validation_type = await asyncio.to_thread(
+            evaluation.run_evaluation,
+            cve_id,
+            patch,
+            language,
+            f"safactory_{request.session_id.replace('-', '_')}",
+            [],
+        )
+    except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)